numpy/_core/tests/test_shape_base.py

```diff
@@ -294,13 +294,15 @@ def test_exceptions(self):
         assert_raises(ValueError, concatenate, ())
 
     @pytest.mark.slow
-    @pytest.mark.skipif(sys.maxsize < 2**32, reason="only problematic on 64bit platforms")
+    @pytest.mark.skipif(sys.maxsize < 2**32,
+                    reason="only problematic on 64bit platforms")
```

Comment:
this looks a bit weird; so how about

```suggestion
    @pytest.mark.skipif(
        sys.maxsize < 2**32,
        reason="only problematic on 64bit platforms"
    )
```

or 

```suggestion
    @pytest.mark.skipif(sys.maxsize < 2**32,
                        reason="only problematic on 64bit platforms")
```